MashUp-iOS/MashUp-iOSTests/Specs/PlatformAttendanceStatusReactorSpec.swift

```diff
@@ -11,62 +11,74 @@ import Nimble
 import Quick
 import RxSwift
 import MashUp_PlatformTeam
+import MashUp_Network
 @testable import MashUp_iOS
 
 final class PlatformAttendanceStatusReactorSpec: QuickSpec {
   override func spec() {
     var sut: PlatformAttendanceStatusReactor!
-    var platformService: PlatformService!
+    var platformService: PlatformServiceMock!
+    let scheduleID = 1
     
-    describe("PlatformAttendanceStatusReactor 로직") {
+    describe("PlatformAttendanceStatusReactor 로직테스트") {
       beforeEach {
-        let repository = mock(PlatformRepository.self)
-        given(repository.attendanceStatus()).willReturn(.just(PlatformAttendanceStatusReactorSpec.mockPlatformAttendance))
-        platformService = PlatformServiceImpl(repository: repository)
-        sut = PlatformAttendanceStatusReactor(platformService: platformService)
+        platformService = mock(PlatformService.self)
+        sut = PlatformAttendanceStatusReactor(platformService: platformService, scheduleID: scheduleID)
       }
       context("리스트에서 iOS클릭시") {
         beforeEach {
           sut.action.onNext(.didTapPlatform(.iOS))
         }
-        it("현재선택플랫폼 iOS로 상태변경") {
+        it("현재선택플랫폼 iOS로 상태변경됩니다.") {
           expect { sut.currentState.selectedPlatform }.to(equal(.iOS))
         }
       }
       context("리스트에서 Android클릭시") {
         beforeEach {
           sut.action.onNext(.didTapPlatform(.android))
         }
-        it("현재선택플랫폼 Android로 상태변경") {
+        it("현재선택플랫폼 Android로 상태변경됩니다.") {
           expect { sut.currentState.selectedPlatform }.to(equal(.android))
         }
       }
       
-      context("진입시 초기정보 가져오기") {
+      context("진입시 초기정보 셋팅을 하면") {
         beforeEach {
+          given(platformService.attendanceStatus(scheduleID: scheduleID)).willReturn(.just(PlatformAttendanceStatusReactorSpec.mockPlatformResponse.asPlatformAttendance))
           sut.action.onNext(.didSetup)
         }
-        it("통신호출") {
-          verify(platformService.attendanceStatus()).wasCalled()
+        
+        it("통신모델을 도메인 모델로 변환시킵니다") {
+          expect { PlatformAttendanceStatusReactorSpec.mockPlatformResponse.asPlatformAttendance }.to(equal(PlatformAttendanceStatusReactorSpec.expactPlatformAttendance))
         }
-        it("플랫폼 리스트 가져옴") {
-          expect { sut.currentState.platformsAttendance }.to(equal(PlatformAttendanceStatusReactorSpec.mockPlatformAttendance.platformAttendanceInformations))
+        it("생성시 넣어준 scheduleID을 가지고 있어야합니다.") {
+          expect { sut.currentState.scheduleID == scheduleID }.to(beTrue())
```

Comment:
```suggestion
          expect { sut.currentState.scheduleID }.to(equal(scheduleID))
```
이쿼터블을 검증하는 경우면 이 메크로가 조금 더 적절할 수 있을 것 같습니답